docs/project-structure.md

```diff
@@ -0,0 +1,39 @@
+## Project Structure
+
+### Basic Monolithic Application Structure
+
+```
+│ .env # Environment credentials
+│ src/
+│├── app.module.ts
+│├── app.controller.ts
+│├── app.service.ts
+│├── modules
+││ ├── module/  # A module that manages a specific domain
+││ │   ├── controllers/ # Responsible for handling incoming requests and returning responses to the client.
+││ │   │   └── module.controller.ts
+││ │   ├── dtos/ DTO is an object that defines strictness as to how data will be sent over the network
+││ │   ├── guards/ # Guards decides a given request will be handled by the route handler or not, depending on certain conditions
+││ │   ├── decorators/ # Custom decorators attaches properties to the request object
+││ │   ├── strategies/ # Implement various authentication mechanisms to handle authentication
```

Comment:
Perhaps we can remove this `strategies`. :thinking: 
Also think if we should remove `guard`.